libs/payments/ui/src/lib/server/purchase-details.tsx

```diff
@@ -6,7 +6,6 @@ import { Invoice } from '@fxa/payments/cart';
 import Image from 'next/image';
 import { formatPlanPricing } from '../utils/helpers';
```

Comment:
Nit: change to `@fxa/payments/ui`

Reply:
Since `formatPlanPricing` is in the same library, relative path here makes sense.